src/hct_mis_api/apps/household/migrations/0020_migration.py

```diff
@@ -0,0 +1,62 @@
+from django.db import migrations
+
+BLANK = ""
+
+def replace_blank_with_null(apps, schema_editor):
+    Household = apps.get_model("household", "Household")
+    Individual = apps.get_model("household", "Individual")
+
+    for instance in Household.objects.all():
```

Comment:
maybe it's better to use `.iterator(chunk_size=1000)` here because qs will be so huge